### PR TITLE
UI update for WebNN Samples and sub pages

### PR DESCRIPTION
**Pull request by ibelem**
@Honry  I updated the UI for the WebNN Samples, PTAL.

- Icon
- Logo
- Font
- Colors

### CODE REVIEW

**Comment by Honry** [comment]
@ibelem Awesome! Could you provide a preview of github page?

File: code/index.html [hunk 1]
```diff
@@ -22,7 +27,7 @@
     <nav class="navbar navbar-expand-md navbar-dark">
       <a href="../index.html">
         <div class="logosvg">
-          <img src="../common/img/logo.svg" />
+          <object data="../common/img/logo.svg"></object>
```
Comment (Honry) [inline code/index.html:30]: The `object`  is not clickable, please add `pointer-events: none;` to its css definition.

**Comment by Honry** [comment]
@ibelem, what's the license of these fonts under `common/css/fonts` folder?

**Comment by ibelem** [comment]
@Honry added License.md for these fonts.

**Open Sans** - https://fonts.google.com/specimen/Open+Sans#license

> These fonts are licensed under the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0).
> You can use them freely in your products & projects - print or digital, commercial or otherwise.
> This isn't legal advice, please consider consulting a lawyer and see the full license for all details.

**Montserrat** - https://fonts.google.com/specimen/Montserrat#license

> These fonts are licensed under the [Open Font License](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL).
> You can use them freely in your products & projects - print or digital, commercial or otherwise.
> This isn't legal advice, please consider consulting a lawyer and see the full license for all details.

**Poppins** - https://fonts.google.com/specimen/Poppins#license

These fonts are licensed under the [Open Font License](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL).
You can use them freely in your products & projects - print or digital, commercial or otherwise.
This isn't legal advice, please consider consulting a lawyer and see the full license for all details.

**Comment by ibelem** [comment]
@Honry  PTAL
https://ibelem.github.io/webnn-samples/index.html
https://ibelem.github.io/webnn-samples/image_classification/index.html

**Review by Honry — APPROVED** [review]
Pretty good, thanks!

**Pull request merged**